custom-recipes/timeseries-forecast-1-train-evaluate/recipe.json

```diff
@@ -292,6 +292,87 @@
             "description": "Statistical models",
             "visibilityCondition": "['choose_algorithms', 'customize_algorithms'].includes(model.forecasting_style)"
         },
+        {
+            "name": "season_length_min",
+            "label": "Season length",
+            "description": "Length of the seasonal period used by statistical models.",
+            "type": "INT",
+            "defaultValue": 1,
+            "mandatory": false,
+            "visibilityCondition": "['choose_algorithms', 'customize_algorithms'].includes(model.forecasting_style) && model.frequency_unit == 'min'"
+        },
+        {
+            "name": "season_length_H",
+            "label": "Season length",
+            "description": "Length of the seasonal period used by statistical models.",
+            "type": "INT",
+            "defaultValue": 24,
+            "mandatory": false,
+            "visibilityCondition": "['choose_algorithms', 'customize_algorithms'].includes(model.forecasting_style) && model.frequency_unit == 'H'"
+        },
+        {
+            "name": "season_length_D",
+            "label": "Season length",
+            "description": "Length of the seasonal period (in days) used by statistical models.",
+            "type": "INT",
+            "defaultValue": 7,
+            "mandatory": false,
+            "visibilityCondition": "['choose_algorithms', 'customize_algorithms'].includes(model.forecasting_style) && model.frequency_unit == 'D'"
+        },
+        {
+            "name": "season_length_B",
+            "label": "Season length",
+            "description": "Length of the seasonal period (in business days) used by statistical models.",
```

Comment:
```suggestion
            "description": "Length of the seasonal period (in days) used by statistical models.",
```

Reply:
for me it's really "in business days". If the seasonality is weekly, then you must select 5 as season length (not 7).

Reply:
Ah yes I see. Sorry, let me close it then